doc/cephfs/cephfs-io-path.rst

```diff
@@ -0,0 +1,29 @@
+ Ceph File System IO Path
+=========================
+
+The Ceph File System (CephFS) stores file data as RADOS objects. File System
+clients access to RADOS directly for reading and writing file data. MDS only
+handle metadata operations.
+
+.. ditaa::
+            +---------------------+
+            |      CephFS FUSE    |
+            +---------------------+
+              |
+              V
+            +---------------------+  Data IOs  +--------------------+
+            |    CephFS Library   | ---------> |       LibRados     |
+            +---------------------+            +--------------------+
+              |                                  |
+              | Metadata Operations              | Objects Read/Write
+              V                                  V
+            +---------------------+            +--------------------+
+            |         MDSs        | -=-------> |         OSDs       |
+            +---------------------+            +--------------------+
+              ^                                  ^
+              | Metadata Operations              | Objects Read/Write
```

Comment:
Can we split this diagram into two, one for the user space client, and the other for the kernel client? I think it will be clearer that way.